web/src/components/AttributeValue/AttributeValue.styled.ts

```diff
@@ -1,11 +1,11 @@
 import {Typography} from 'antd';
 import styled, {css} from 'styled-components';
 
-export const ValueText = styled(Typography.Text)<{isCollapsed: boolean}>`
+export const ValueText = styled(Typography.Text)<{$isCollapsed: boolean}>`
   cursor: pointer;
 
-  ${({isCollapsed}) =>
-    isCollapsed
+  ${({$isCollapsed}) =>
```

Comment:
What's the idea around the $? is this causing any warnings or something like that?

Reply:
https://styled-components.com/docs/api#transient-props